mappings/net/minecraft/client/util/Session.mapping

```diff
@@ -3,19 +3,30 @@ CLASS net/minecraft/class_320 net/minecraft/client/util/Session
 	FIELD field_1983 accessToken Ljava/lang/String;
 	FIELD field_1984 accountType Lnet/minecraft/class_320$class_321;
 	FIELD field_1985 uuid Ljava/lang/String;
+	FIELD field_34960 xuid Ljava/util/Optional;
+	FIELD field_34961 clientId Ljava/util/Optional;
 	METHOD <init> (Ljava/lang/String;Ljava/lang/String;Ljava/lang/String;Ljava/util/Optional;Ljava/util/Optional;Lnet/minecraft/class_320$class_321;)V
 		ARG 1 username
 		ARG 2 uuid
 		ARG 3 accessToken
+		ARG 4 xuid
+		ARG 5 clientId
+		ARG 6 accountType
 	METHOD method_1673 getUuid ()Ljava/lang/String;
 	METHOD method_1674 getAccessToken ()Ljava/lang/String;
 	METHOD method_1675 getSessionId ()Ljava/lang/String;
 	METHOD method_1676 getUsername ()Ljava/lang/String;
 	METHOD method_1677 getProfile ()Lcom/mojang/authlib/GameProfile;
 	METHOD method_35718 getAccountType ()Lnet/minecraft/class_320$class_321;
+	METHOD method_38740 getClientId ()Ljava/util/Optional;
+	METHOD method_38741 getXuid ()Ljava/util/Optional;
```

Comment:
What's the source of this "xuid"? They just call it `UserId` in string literals.

Reply:
It is what it is refer to in the blog post:
https://www.minecraft.net/en-us/article/minecraft-snapshot-21w38a

Reply:
Hmm, a search shows it probably means xbox user id. I'd still call it `UserId` because I haven't seen xuid in the literal strings

Reply:
It's weird. The command argument defined in `Main` is `xuid`, and the name in other places are `userId`. Maybe I should rename.

Reply:
Renamed param to `userId` to reduce confusion (and just in case they want to support mojang account). I think we should still call it xuid in `Session` because 1) that's the name of the cmd argument that is passed to init and 2) other account types may have other types of user id.